[2024-09-29T12:45:04.458+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-29T12:45:04.534+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v04.postgres_to_s3 scheduled__2022-08-05T00:00:00+00:00 [queued]>
[2024-09-29T12:45:04.599+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v04.postgres_to_s3 scheduled__2022-08-05T00:00:00+00:00 [queued]>
[2024-09-29T12:45:04.607+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-29T12:45:04.670+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): postgres_to_s3> on 2022-08-05 00:00:00+00:00
[2024-09-29T12:45:04.703+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=22509) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-29T12:45:04.706+0000] {standard_task_runner.py:72} INFO - Started process 22527 to run task
[2024-09-29T12:45:04.704+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks_v04', 'postgres_to_s3', 'scheduled__2022-08-05T00:00:00+00:00', '--job-id', '992', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_hooks.py', '--cfg-path', '/tmp/tmpv6nmer1p']
[2024-09-29T12:45:04.707+0000] {standard_task_runner.py:105} INFO - Job 992: Subtask postgres_to_s3
[2024-09-29T12:45:04.763+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-09-29T12:45:04.867+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_postgres_hooks_v04.postgres_to_s3 scheduled__2022-08-05T00:00:00+00:00 [running]> on host 6851fee35b72
[2024-09-29T12:45:04.870+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-09-29T12:45:05.309+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks_v04' AIRFLOW_CTX_TASK_ID='postgres_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2022-08-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-08-05T00:00:00+00:00'
[2024-09-29T12:45:05.330+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-29T12:45:05.423+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/operator_helpers.py:175 AirflowContextDeprecationWarning: Accessing 'next_ds_nodash' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds_nodash }}' instead.
[2024-09-29T12:45:05.462+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-09-29T12:45:05.486+0000] {dag_with_postgres_hooks.py:34} INFO - Saved orders data in text file: dags/get_orders_20220805.txt
[2024-09-29T12:45:05.517+0000] {base.py:84} INFO - Retrieving connection 'minio_conn'
[2024-09-29T12:45:05.520+0000] {connection_wrapper.py:391} INFO - AWS Connection (conn_id='minio_conn', conn_type='aws') credentials retrieved from extra.
[2024-09-29T12:45:05.522+0000] {logging_mixin.py:190} WARNING - <string>:9 AirflowProviderDeprecationWarning: extra['host'] is deprecated and will be removed in a future release. Please set extra['endpoint_url'] instead
[2024-09-29T12:45:07.798+0000] {dag_with_postgres_hooks.py:43} INFO - Orders file /tmp/tmpk_qx3x0120220805 has been pushed to S3!
[2024-09-29T12:45:07.811+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-29T12:45:07.897+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-29T12:45:07.904+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks_v04, task_id=postgres_to_s3, run_id=scheduled__2022-08-05T00:00:00+00:00, execution_date=20220805T000000, start_date=20240929T124504, end_date=20240929T124507
[2024-09-29T12:45:08.221+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-29T12:45:08.498+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
